.nodes.sh

```diff
@@ -1,7 +1,3 @@
 #!/bin/bash
 
-HN=$(hostname)
-
-if [ "$HN" == "ccf-perf-uk-0" ]; then
-    echo "-n ssh://10.1.0.7 -n ssh://10.1.0.4 -n ssh://10.1.0.5"
-fi
\ No newline at end of file
+echo "-n local://localhost -n local://localhost"
```

Comment:
This script was necessary when we had custom logic for certain nodes, but I think now it would be simpler to just have these `-n` arguments directly in the yaml, and remove this file?

Reply:
Sure, done.